Metalama.Framework/Advising/IAdviceFactory.cs

```diff
@@ -148,6 +135,15 @@ public interface IAdviceFactory
         ///     of the <see cref="meta"/> API.</param>
         IOverrideAdviceResult<IConstructor> Override( IConstructor targetConstructor, string template, object? args = null, object? tags = null );
 
+        IIntroductionAdviceResult<IConstructor> IntroduceConstructor(
```

Comment:
This method should be documented.

Reply:
This is WIP, it will be finished in constructor introduction story.